collections/_programs/hacktoberfest_2020.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: project
+title: "Hacktoberfest"
+image:
+path: /assets/images/buttons/hf-icon.png
+thumbnail: /assets/images/buttons/hf-icon.png
+link: /programs/hacktoberfest
+permalink: /programs/hacktoberfest
+excerpt: "Layer5 - Hacktoberfest"
+---
+<style>
+  div.gsoc {
+    margin-left: 15px;
+  }
+
+  li.gsoc {
+    margin-left: 15px;
+  }
+</style>
+<div style="z-index: 20;">
+  <h3 class="black-text">Hacktoberfest 2020 and Layer5</h3>
+  Hacktoberfest is open to everyone in our global community. Whether you’re a developer, student learning to code, event host, or company of any size, you can help drive growth of open source and make positive contributions to an ever-growing community. All backgrounds and skill levels are encouraged to complete the challenge. You can find more information here: <a href="https://hacktoberfest.digitalocean.com">https://hacktoberfest.digitalocean.com</a>
+ <ul>
+    <ol>
+      <b> Hacktoberfest is a celebration open to everyone in our global community.</b>
+    </ol>
+    <ol>
+      While small, the Layer5 community represents the largest collection of
+      service mesh projects and their maintainers in the world. We build
+      projects to provide learning environments, deployment and operational best
+      practices, performance benchmarks, create documentation, share networking
+      opportunities, and more. Our shared commitment to the open source spirit
+      pushes Layer5 projects forward. New members are always welcome.
+    </ol>
+    <ol>
+      <b> Layer5 has a long list of interesting issues coming up for our Hacktoberfest participants! </b>
+    </ol>
+    <ol>
+      Find a list of beginner friendly issues here: <a href="https://github.com/search?q=org%3Alayer5io+label%3Ahacktoberfest&type=issues"> https://github.com/search?q=org%3Alayer5io+label%3Ahacktoberfest&type=issues </a> to make your mark in open-source and move towards successfully completing the Hacktoberfest challenge.
+    </ol>
+    <ol>
+      <b> Fill the Questionnaire about Open Source </b>
+    </ol>
+    <ol>
+      Layer5 is a warm and welcoming community of open source contributors. We are continuously welcoming new members and are looking for your input. 
+
+      Help us better prepare to support you as you find your way into the world of Open Source by filling up this questionnaire: <a href="https://docs.google.com/forms/d/e/1FAIpQLSeQmQv6Xrnb69qGUZjHcw6i92jNDXir6WN6sRAHDNRtKUzBUg/viewform">https://docs.google.com/forms/d/e/1FAIpQLSeQmQv6Xrnb69qGUZjHcw6i92jNDXir6WN6sRAHDNRtKUzBUg/viewform</a>
```

Comment:
Hi @jesuslerma could you just add this Hacktoberfest Meetup event link to this page as well 
`https://organize.mlh.io/participants/events/4796-getting-started-layer5`